schemas/glean/baseline/baseline.1.parquetmr.txt

```diff
@@ -0,0 +1,163 @@
+message baseline {
+  required group metadata {
+    optional binary EnvVersion (UTF8);
+    optional binary geoSubdivision1 (UTF8);
+    optional binary geoSubdivision2 (UTF8);
+    optional int32 Severity;
+    required binary Host (UTF8);
+    required binary Hostname (UTF8);
+    required binary Logger (UTF8);
+    required binary Type (UTF8);
+    required binary docType (UTF8);
+    required binary documentId (UTF8);
+    required binary geoCity (UTF8);
+    required binary geoCountry (UTF8);
+    required double docVersion;
+    required fixed_len_byte_array(16) Uuid;
+    required int32 Pid;
+    required int64 Timestamp;
+  }
+  required group ping_info {
+    required binary ping_type (UTF8);
+    required binary app_build (UTF8);
+    required binary telemetry_sdk_build (UTF8);
+    required binary client_id (UTF8);
+    required int32 seq;
+    required binary start_time (UTF8);
+    required binary end_time
+  }
+  optional group metrics {
+    optional group boolean (MAP) {
+      repeated group key_value {
+        required binary key (UTF8);
+        optional boolean value;
+      }
+    }
+    optional group string (MAP) {
+      repeated group key_value {
+        required binary key (UTF8);
+        optional binary value (UTF8);
+      }
+    }
+    optional group number (MAP) {
+      repeated group key_value {
+        required binary key (UTF8);
+        optional double value;
```

Comment:
64-bit floating point -- is this appropriate for the `number` type in the glean SDK?

Reply:
Yes, I think so.  Other "numeric-like" types, like count, will be integers, but I think this one is sort of free-form/general purpose, so 64-bit float is probably the right choice.  (But we haven't implemented this one yet).